src/main/java/frc/trigon/robot/subsystems/arm/armConstants.java

```diff
@@ -0,0 +1,65 @@
+package frc.trigon.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.sensors.AbsoluteSensorRange;
+import com.ctre.phoenix.sensors.CANCoder;
+import com.ctre.phoenix.sensors.CANCoderConfiguration;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.math.controller.PIDController;
+
+public class armConstants {
```

Comment:
Class names should be CamelCase. See [Java Naming Conventions](https://www.thoughtco.com/using-java-naming-conventions-2034199)